APickup/APickup.h

```diff
@@ -44,7 +44,7 @@ class VRBASE_API APickup : public ABaseEntity {
 
             m_pPickupMeshComponent->SetStaticMesh(m_pStaticMesh);
 
-            if (PropertyName == FName(STRINGIZE(m_pStaticMesh)))
+            if (PropertyName == FName(toString(m_pStaticMesh)))

```

Comment:
I renamed `STRINGIZE` to `toString` for clarity.

Ideally, for maximum compatibility with our C# projects, we should try to reduce macro usage as much as possible.